src/AutoRest.CSharp/Generation/Writers/ArmResourceWriter.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using AutoRest.CSharp.Output.Models;
+
+namespace AutoRest.CSharp.Generation.Writers
+{
+    internal class ArmResourceWriter
```

Comment:
Should we start grouping writers?

Reply:
are we going to group / separate lowlevel vs dataplane?  If so then it might make sense to declare what are the top level types and group all files accordingly including a common?  We can take this up in a future refactoring once we see how it ends up